_inc/client/components/data/query-site-benefits/index.jsx

```diff
@@ -0,0 +1,35 @@
+/**
+ * External dependencies
+ */
+import { Component } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { fetchSiteBenefits, isFetchingSiteBenefits } from 'state/site';
+
+class QuerySiteBenefits extends Component {
+	static defaultProps = {
+		isFetchingSiteBenefitsData: false,
+	};
+
+	UNSAFE_componentWillMount() {
```

Comment:
Could this be done in some other way than introducing legacy methods in a new code? Otherwise, you're just punting this work for anyone who attempts to update React in the future.

https://reactjs.org/docs/react-component.html#unsafe_componentwillmount

Reply:
I mindlessly copied this component in from another PR 🤦‍♂️. I will avoid having a PR with `UNSAFE_x` in it 👍